patches/email-verification-for-woocommerce-pro.0002.fix.resend-email-nonce-is-not-enforced.patch.patch

```diff
@@ -0,0 +1,33 @@
+From b18d3e32df3bdb7d973b285b0f5b6d8ab9abee7a Mon Sep 17 00:00:00 2001
```

Comment:
Duplicate .patch extension in filename.

Reply:
Also try to name the patch files from a higher level perspective - less words, more context:
```
fix.security-resend-email-nonce
```